web/viewer.css

```diff
@@ -1509,7 +1509,8 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   display: none;
 }
 
-@media screen and (min-resolution: 2dppx) {
+@media screen and (min-resolution: 2dppx),
+       screen and (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
@Snuffleupagus, I don't think your suggestion is valid CSS (and it's not accepted by browsers).

The at-rule syntax is described here:

  http://www.w3.org/TR/css-syntax-3/#consume-an-at-rule0

  "The @media rule is a conditional group rule whose condition is a media query. It consists of the at-keyword '@media' followed by a (possibly empty) media query list"

And the @media semantics are described here:

  http://dev.w3.org/csswg/css-conditional-3/#at-ruledef-media

If the CSS preprocessor in use doesn't support at-rules I guess all bets are off for this pull request. (Is it confirmed not-working at your end?)